README.md

```diff
@@ -22,27 +22,23 @@ CNFs. An example of a custom VPP-based CNF is the [Contiv-VPP][contiv-vpp] vswit
 |Release|Release Date|Info|
 |---|:---:|---|
 |[![stable](https://img.shields.io/github/release/ligato/vpp-agent.svg?label=release&logo=github)](https://github.com/ligato/vpp-agent/releases/latest)|![Release date](https://img.shields.io/github/release-date/ligato/vpp-agent.svg?label=)|latest release|
-|[![latest](https://img.shields.io/github/release-pre/ligato/vpp-agent.svg?label=release&logo=github)](https://github.com/ligato/vpp-agent/releases)|![Release date](https://img.shields.io/github/release-date-pre/ligato/vpp-agent.svg?label=)|last release/pre-release|
 
 Have a look at the [release notes](CHANGELOG.md) for a complete list of changes.
 
 ### Branches
 
 |Branch|Last Commit|Info|
-|---|:---:|---|
-|[![master](https://img.shields.io/badge/branch-master-blue.svg?logo=git&logoColor=white)](https://github.com/ligato/vpp-agent/tree/master)|![GitHub last commit (branch)](https://img.shields.io/github/last-commit/ligato/vpp-agent/master.svg?label=)| has **moved to v2**, introducing several [breaking changes](https://github.com/ligato/vpp-agent/blob/master/CHANGELOG.md#v200) :warning:|
-|[![dev](https://img.shields.io/badge/branch-dev-green.svg?logo=git&logoColor=white)](https://github.com/ligato/vpp-agent/tree/dev)|![GitHub last commit (branch)](https://img.shields.io/github/last-commit/ligato/vpp-agent/dev.svg?label=)|will be used for all the future **development**|
-
-**Note:** The branch pantheon-dev has been **deprecated** (v1).
```

Comment:
I saw that you've added L3XC feature directly to master. Doesn't this mean that now dev should be in this note?

Reply:
It is listed with other branches.